README.md

```diff
@@ -410,6 +423,17 @@ Another attack is where `evil.com` communicates with other third parties by sett
 
 One way to potentially circumvent this is to make the global per-partition limit much larger than the per-partition limit for each third-party domain, but it is unclear what the relative size of the global per-partition limit would have to be to mitigate these attacks.
 
+### Applying the 180 cookies-per-domain limit
+
+We recommend that user agents do not include `Partitioned` cookies in the 180 per-domain cookie limit.
+The reason being that malicious third parties could abuse this information to learn information about users across different top-level partitions.
+
+Say `evil.com` was embedded on a site, `1p.com`, and it sets 180 cookies on the users' machines when they visit `1p.com`.
+On other top-level sites, `evil.com` sets another cookie which evicts one of the cookies `evil.com` set in `1p.com`'s partition.
+When a user returns to `1p.com`, `evil.com` can look at how many cookies were evicted from its partition on `1p.com` to see if the user visited another site with `evil.com`.
+
+This attack can happen even if user agents enforce limitations on the number of cookies, since `evil.com` can just set as many cookies as it is allowed per-partition and then observe if any are evicted. It is not clear what the relative limits need to be to prevent `evil.com` from learning any identifiable information about users this way.
```

Comment:
I feel like something is missing in the phrasing here. I don't understand the idea as it's written. Did you mean "even if user agents enforce limitations on the number of cookies *per-partition*"? If so, I'm not sure what the attack is, since stuffing one partition with cookies doesn't affect another partition?

Reply:
Oops. Fixed the wording.

Say there is a global 180 per-domain cookie limit, and a limit for 3Ps per-partition as well. I am not sure if malicious 3Ps can still take advantage of the former limit to learn cross-site behavior of users. I think the latter limit makes it harder for them, but I'm not convinced if it fully mitigates the attack.